lighthouse-viewer/app/src/main.js

```diff
@@ -7,32 +7,7 @@
 
 /* global LighthouseReportViewer, Logger */
 
-/**
- * @param {string} src
- * @return {Promise}
- */
-function loadScript(src) {
-  return new Promise((resolve, reject) => {
-    const script = document.createElement('script');
-    script.src = src;
-    script.onload = resolve;
-    script.onerror = reject;
-    // document.head is defined.
-    /** @type {HTMLHeadElement} */ (document.head).appendChild(script);
-  });
-}
-
-const loadPolyfillPromises = [];
-if (!('fetch' in window)) {
-  loadPolyfillPromises.push(loadScript('./src/polyfills/fetch.js'));
-}
-if (!('URLSearchParams' in window)) {
-  loadPolyfillPromises.push(loadScript('./src/polyfills/url-search-params.js'));
-}
-
-// Lazy load polyfills that are needed. If any of the load promises fails,
-// stop and don't create a report.
-Promise.all(loadPolyfillPromises).then(_ => {
+function main() {
```

Comment:
cheeky little trick to not lose any blame. it's a main script so it deserves a main fn.